spec/index.test.js

```diff
@@ -41,33 +30,57 @@ describe('Database Tests', () => {
     await Photos.create(mockData);
 
     server = await app.listen(4000, (err) => {
-    if (err) {
-     return done(err);
-    }
+      if (err) {
+       return done(err);
+      }
 
-    agent = request.agent(server); // since the application is already listening, it should use the allocated port
-    done();
+      agent = request.agent(server); // since the application is already listening, it should use the allocated port
+      done();
     });
   });
+
   afterAll(async function(done){
     await mongoose.disconnect();
     return server && await server.close(done);
   });
 
-  it('Should retrieve data from Photos database', function(done) {
+  test('Should retrieve the correct amount of dcouments from the database', function(done) {
     Photos.countDocuments({}, (err, items) => {
       if(err) {throw err;}
-      expect(items).equal(2);
+      expect(items).to.equal(2);
       done();
     });
   });
 
-  it('Should retrieve the correct amount of dcouments from the database', function(done) {
-    Photos.countDocuments({}, (err, items) => {
-      if(err) {throw err;}
-      expect(items).equal(2);
-      done();
-    });
+  test('Should respond with an error to invalid paths', async () => {
+    const res = await request(server).get('/stays');
+    expect(res.statusCode).to.equal(404);
+  });
+
+  test('Should respond with a 405 Method Not Allowed to POST requests to /api/rooms/:roomId/', async () => {
+    const res = await request(server).post('/api/rooms/4160333/');
+    expect(res.statusCode).to.equal(405);
+  });
+
+  test('Should respond with a 405 Method Not Allowed to PUT requests to /api/rooms/:roomId/', async () => {
+    const res = await request(server).put('/api/rooms/4160333/');
+    expect(res.statusCode).to.equal(405);
+  });
+
+  test('Should respond with a 405 Method Not Allowed to PATCH requests to /api/rooms/:roomId/', async () => {
+    const res = await request(server).patch('/api/rooms/4160333/');
+    expect(res.statusCode).to.equal(405);
+  });
+
+  test('Should respond with a 405 Method Not Allowed to DELETE requests to /api/rooms/:roomId/', async () => {
+    const res = await request(server).delete('/api/rooms/4160333/');
+    expect(res.statusCode).to.equal(405);
+  });
+
+  test('Should correct amount of photo records from DB on GET requests to /api/rooms/:roomId/', async () => {
+    const res = await request(server).get('/api/rooms/4160333');
+    expect(res.body[0].roomId).to.equal(4160333);
+    expect(res.body.length).to.equal(2);
```

Comment:
Let's update these endpoints to match the updates you're making in the `apiRouter`